### ISSUE TITLE
 403 when trying to retrieve

### ISSUE THREAD

**Issue by inteist** [firebase/quickstart-nodejs#22]
I was getting:

> Unable to get template.
undefined

so I examined the code and modified it to:
```js
        console.log("Unable to get template.");
        console.log(resp.statusCode);
        console.log(resp.error);
```

to see the status code at least since `resp.error` is empty and `resp` is very large.
 
and now I get:
> Unable to get template.
403
undefined

I have the firebase-admin for the database working so I know that my PK should be working. I thought I may only have it enabled for the realtime database but I couldn't figure out where to see info on that or how to modify it.

**Comment by inteist** [comment]
as always, just after I posted I found the solution.

I have googled `firebaseremoteconfig.googleapis.com 403` and found this page:
https://firebase.google.com/docs/remote-config/use-config-rest

with the following instructions:

For Firebase projects created before the March 7, 2018 release of the Remote Config REST API, you must enable the API in the Google APIs console.

1. Open the Firebase Remote Config API page (https://console.developers.google.com/apis/api/firebaseremoteconfig.googleapis.com/overview?project=) in the Google APIs console.
2. When prompted, select your Firebase project. (Every Firebase project has a corresponding project in the Google APIs console.)
3. Click Enable on the Firebase Remote Config API page.
